impress-org/givewp
# Refactor: add telemetry support for v3 forms

Related PR: https://github.com/impress-org/give-form-field-manager/pull/405

## Description



This PR adapts the already existing code to also send v3 form data to the telemetry server.

This PR doesn't track any new data, it just sends the same data used on V2 forms to V3 forms as well. But as the data for both are stored differently, was necessary to check and prepare the data according to the form version.

To do that, was necessary to register on the new `givewp_form_builder_updated` hook a function to track the data for the v3 forms. 

## Affects



The tracking data for v3 forms.

## Testing Instructions



1. Install it on a live server; 
2. Active the tracking option on the settings > advanced page;
3. Edit a V3 form;
4. Make sure no fatal errors are thrown in the logs. 

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205376382458388
  - https://app.asana.com/0/0/1205184750257818
